app/stac_api/serializers.py

```diff
@@ -520,14 +518,15 @@ def update_or_create(self, look_up, validated_data):
         asset, created = Asset.objects.update_or_create(**look_up, defaults=validated_data)
         return asset, created
 
+    def validate_type(self, value):
```

Comment:
Maybe add a comment that this method validate the field `type`. `validate_type()` is so generic that it could be misunderstood.